string data
float64 superFrameNumber
